tutorials/18_diagnostics_lc2st.ipynb

```diff
@@ -0,0 +1,821 @@
+      "text/plain": [
+       "<Figure size 1200x300 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, axes = plt.subplots(1,len(thetas_star), figsize=(12,3))\n",
+    "for i in range(len(thetas_star)):\n",
+    "    probs, scores = lc2st.get_scores_on_observed_data(\n",
+    "        theta_o=post_samples_star[i],\n",
+    "        x_o=xs_star[i],\n",
+    "        return_probs=True\n",
+    "    )\n",
+    "    T_data = lc2st.get_statistic_on_observed_data(\n",
+    "        theta_o=post_samples_star[i],\n",
+    "        x_o=xs_star[i]\n",
+    "    )\n",
+    "    T_null = lc2st.get_statistics_under_null_hypothesis(\n",
+    "        theta_o=post_samples_star[i],\n",
+    "        x_o=xs_star[i]\n",
+    "    )\n",
+    "    p_value = lc2st.p_value(post_samples_star[i], xs_star[i])\n",
+    "    reject = lc2st.reject_test(post_samples_star[i], xs_star[i], alpha=conf_alpha)\n",
+    "\n",
+    "    # plot 95% confidence interval\n",
+    "    quantiles = np.quantile(T_null, [0, 1-conf_alpha])\n",
+    "    axes[i].hist(T_null, bins=50, density=True, alpha=0.5, label=\"Null\")\n",
+    "    axes[i].axvline(T_data, color=\"red\", label=\"Observed\")\n",
+    "    axes[i].axvline(quantiles[0], color=\"black\", linestyle=\"--\", label=\"95% CI\")\n",
+    "    axes[i].axvline(quantiles[1], color=\"black\", linestyle=\"--\")\n",
+    "    axes[i].set_xlabel(\"Test statistic\")\n",
+    "    axes[i].set_ylabel(\"Density\")\n",
+    "    axes[i].set_xlim(-0.01,0.25)\n",
+    "    axes[i].set_title(\n",
+    "        f\"observation {i+1} \\n p-value = {p_value:.3f}, reject = {reject}\"\n",
+    "    )\n",
+    "axes[i].legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Qualitative diagnostics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### PP-Plots\n",
+    "\n",
+    "PP-plots allow to evaluate a general trend of over or under confidence, by comparing the predicted probabilities of belonging to the estimated posterior (class 0). If the red curve is not fully contained in the gray confidence region, this means that the test rejects the null hypothesis and that significant deviances from the true postrior are detected. Here two scenarios are possible:\n",
```

Comment:
```suggestion
    "P-P plots allow to evaluate a general trend of over- or under-confidence, by comparing the predicted probabilities of belonging to the estimated posterior (class 0). If the red curve is not fully contained in the gray confidence region, this means that the test rejects the null hypothesis and that a significant discrepancy from the true posterior is detected. Here two scenarios are possible:\n",
```

Reply:
I suggest to make the statement at the end more strong. discrepancy can do the job for me.